src/Morpho.sol

```diff
@@ -469,8 +469,9 @@ contract Morpho is IMorphoStaticTyping {
     /// @dev Accrues interest for the given market `marketParams`.
     /// @dev Assumes that the inputs `marketParams` and `id` match.
     function _accrueInterest(MarketParams memory marketParams, Id id) internal {
-        uint256 elapsed = block.timestamp - market[id].lastUpdate;
+        if (marketParams.irm == address(0)) return;
```

Comment:
Ideally it should, but we currently never emit `lastUpdate` and expect the offchain integrator to hardcode that `lastUpdate = block.timestamp` every time a function that accrues interest is called

Reply:
if we can simplify the logic here we should do it: that's why we need #642

Reply:
So you take care of it in #642?
I think it's better as the code is consistent in its current state, and the spec may be changed in #642

Reply:
>I think it's better as the code is consistent in its current state

to me no because now there are some markets where lastUpdated is never updated, and the AccruedInterest event is never emitted. It can be part of the spec for sure, but it complexifies the spec for nothing imo

Reply:
> now there are some markets where lastUpdated is never updated

makes me think that it saves a lot of gas

> It can be part of the spec for sure, but it complexifies the spec for nothing IMO

So I wouldn't say "for nothing" ^^

Reply:
2000 something, I think that it's not worth complexifying the spec for that.

Reply:
Isn't it 5k to SSTORE a cold slot?

Reply:
not for a hot slot

Reply:
But how is it hot? it should never been touched ever with this design

Reply:
Because of 

`require(market[id].lastUpdate != 0, ErrorsLib.MARKET_NOT_CREATED);`